bamboo/deploy-to-npm.sh

```diff
@@ -14,6 +14,9 @@ export NPM_TAG=$(node ./bamboo/npm-tag.js);
 echo "Publishing packages to NPM with version=${VERSION} and tag=${NPM_TAG}"
 export npm_config_unsafe_perm=true
 
+npx lerna run package
```

Comment:
Running both to ensure future deployments have always run both.      We should eventually remove prepare once the typescript migration/changes are working.